templates/vpc/content/outputs.tf

```diff
@@ -0,0 +1,43 @@
+output "vpc_id" {
```

Comment:
Minor nit: could you run `terraform fmt` on the `templates/vpc` folder

Reply:
Sorry about this! it has been ran.

Reply:
Thank you!